source/index.rst

```diff
@@ -133,4 +130,3 @@ Open edX Documentation
 
       other/feedback
       other/getting_help
```

Comment:
Is the missing newline on purpose?

Reply:
Yea, there's still a newline, just not an empty line after that.